webapp/channels/src/actions/views/create_comment.tsx

```diff
@@ -146,6 +175,35 @@ export function makeOnSubmit(channelId: string, rootId: string, latestPostId: st
     };
 }
 
+export function onSubmit(draft: PostDraft, options: {ignoreSlash?: boolean}): ActionFuncAsync<boolean, GlobalState> {
+    return async (dispatch, getState) => {
+        const {message, channelId, rootId} = draft;
+        const state = getState();
+
+        dispatch(addMessageIntoHistory(message));
+
+        // const key = rootId ? `${StoragePrefixes.COMMENT_DRAFT}${rootId}` : `${StoragePrefixes.DRAFT}${channelId}`;
+        // dispatch(removeDraft(key, channelId, rootId));
```

Comment:
do we want to leave this commented?

Reply:
No, it can be removed. Probably I was making sure the logic was deleting the draft correctly. The draft should be removed by code on `useSubmit`.